csharp/ql/lib/change-notes/2023-06-06-dotnettest.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* C#: Update the LUA tracer such that `-p:SharedCompilation=false` is not injected when `dotnet test` is applied to a `dll` or `exe` file.
```

Comment:
I tried to reword this to focus more on the effect of the change than the implementation. How does this sound?
```suggestion
* C#: Analysis of the `dotnet test` command supplied with a `dll` or `exe` file as argument no longer fails due to the addition of an erroneous `-p:SharedCompilation=false` argument.
```